src/components/Button/Button.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import confirm from "../../assets/confirm.svg";
+import remove from "../../assets/remove.svg";
+
+export const Button = ({ text, icon, click, iconSize = 8 }) => {
+  const IconSize = `h-${iconSize} w-${iconSize}`;
```

Comment:
Be careful keeping 2 variables with different cases: `iconSize` and `IconSize`. You definitely should rename the it to `iconClassName`. Simple